whylog/constraints/verifier.py

```diff
@@ -0,0 +1,110 @@
+from whylog.constraints import IdenticalIntervals, TimeConstraint
+from whylog.exceptions import WhylogError
+from whylog.front import FrontInput
+
+
+class ConstraintManager(object):
+    @classmethod
+    def get_constraint_by_type(cls, constraint_data):
+        constraints = {
+            'identical': IdenticalIntervals({}),
+            'time': TimeConstraint({})
+            # register your constraint here
+        }
+        if constraint_data['name'] in constraints:
+            return constraints[constraint_data['name']]
+        raise WhylogError("No such constraint (%s) registered" % constraint_data['name'])
```

Comment:
There should be raise UnsupportedConstraintType or something like that, what is subclass of WhylogError